environments/network/demo.tfvars

```diff
@@ -0,0 +1,49 @@
+enable_debug = "true"
+
+network_address_space                  = "10.50.64.0/18"
+aks_00_subnet_cidr_blocks              = "10.50.64.0/20"
+aks_01_subnet_cidr_blocks              = "10.50.80.0/20"
+iaas_subnet_cidr_blocks                = "10.50.96.0/24"
+application_gateway_subnet_cidr_blocks = "10.50.97.0/25"
+
+additional_subnets = [
```

Comment:
can you add api-subnet as per https://github.com/hmcts/aks-cft-deploy/blob/main/environments/network/ithc.tfvars#L10-L13 ? 

Give it IP address: 10.50.97.128/25

Reply:
added this in.